downstream/modules/platform/ref-converting-playbook-examples.adoc

```diff
@@ -0,0 +1,189 @@
+
+.Examples
+
+This example is of a shared directory called */mydata* in which we want to be able to read and write files to during a job run. Remember this has to already exist on the execution node we will be using for the automation run.
+
+You will target the _aape1.local_ execution node to run this job, because the underlying hosts already has this in place:
+
+----
+[awx@aape1 ~]$ ls -la /mydata/
+total 4
+drwxr-xr-x.  2 awx  awx   41 Apr 28 09:27 .
+dr-xr-xr-x. 19 root root 258 Apr 11 15:16 ..
+-rw-r--r--.  1 awx  awx   33 Apr 11 12:34 file_read
+-rw-r--r--.  1 awx  awx    0 Apr 28 09:27 file_write
+----
+
+You will use a simple playbook to launch the automation with sleep defined to allow you access, and to understand the process, as well as demonstrate reading and writing to files.
+
+----
+# vim:ft=ansible:
+----
+
+----
+- hosts: all
+  gather_facts: false
+  ignore_errors: yes
+  vars:
+    period: 120
+    myfile: /mydata/file
+  tasks:
+    - name: Collect only selected facts
+      ansible.builtin.setup:
+        filter:
+          - 'ansible_distribution'
+          - 'ansible_machine_id'
+          - 'ansible_memtotal_mb'
+          - 'ansible_memfree_mb'
+    - name: "I'm feeling real sleepy..."
+      ansible.builtin.wait_for:
+        timeout: "{{ period }}"
+      delegate_to: localhost
+    - ansible.builtin.debug:
+        msg: "Isolated paths mounted into execution node: {{ AWX_ISOLATIONS_PATHS }}"
+    - name: "Read pre-existing file..."
+      ansible.builtin.debug:
+        msg: "{{ lookup('file', '{{ myfile }}_read'
+    - name: "Write to a new file..."
+      ansible.builtin.copy:
+        dest: "{{ myfile }}_write"
+        content: |
+          This is the file I've just written to.
+          
+    - name: "Read written out file..."
+      ansible.builtin.debug:
+        msg: "{{ lookup('file', '{{ myfile }}_write') }}"
+
+----
+
+In Ansible Automation Platform 2 from the Settings menu, select menu:Job Settings[Jobs]. 
+
+Paths to expose isolated jobs:
+
+----
+[
+"/mydata:/mydata:rw"
+]
+----
+
+The volume mount is mapped with the same name in the container and has read-write capability. This will get used when you launch the job template.
+
+The _prompt on launch_ should be set for _extra_vars_ so you can adjust the sleep duration for each run, The default is 30 seconds.
+
+Once launched, and the _wait_for_ module is invoked for the sleep, you can go onto the execution node and look at what is running.   
+
+To verify the run has completed successfully, run this command to get an output of the job:
+
+----
+$ podman exec -it 'podman ps -q' /bin/bash
+bash-4.4#
+----
+
+You are now inside the running execution environment container.
+
+Look at the permissions, you will see that *awx* has become ‘root’, but this is not really root as in the superuser, as you are using rootless Podman, which maps users into a kernel namespace similar to a sandbox. Learn more about How does rootless Podman work? for shadow-utils.
```

Comment:
Do we need a cross-reference here?

Reply:
Yes and no. This reference is in the reference module.  We do not need an xref due to this being and example of what can happen when running jobs under SELinux.